crates/matrix-sdk-base/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ All notable changes to this project will be documented in this file.
 - Replaced `Room::compute_display_name` with the reintroduced `Room::display_name()`. The new
   method computes a display name, or return a cached value from the previous successful computation.
   If you need a sync variant, consider using `Room::cached_display_name()`.
+- [**breaking**]: The reexported types `SyncTimelineEvent` and `TimelineEvent` have been fused into a single type
```

Comment:
```suggestion

- [**breaking**]: The reexported types `SyncTimelineEvent` and `TimelineEvent` have been fused into a single type
```

Reply:
I'm not against this change, but can I ask why we should do it like this? This will break the list into two lists, now, which seems counter to the point of having *a* list of changes.

Reply:
I don't particularly care, but if you prefer the rendering without the extra newlines, then we shouldn't have them in the other changelog either.